src/core/examples.js

```diff
@@ -63,7 +63,7 @@ export function run(conf) {
       example.classList.remove("example", "illegal-example");
       // relocate the id to the div
       const id = example.id ? example.id : null;
-      if (id !== "") {
+      if (id !== "" && id !== null) {
```

Comment:
`""` isn't really needed I guess.  `example.id ? ... ` will be `true` only if `id` is non-empty

Reply:
yeah, changed it.